swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=None, age=None, name=None, ):
+        self.id = id if id else uuid.uuid4().int
```

Comment:
Something to consider, what happens here if someone tries to pass in an `id` that is not `None`, but is an `id` that is already assigned to something in inventory? What type of logic could we implement here in order to prevent that from happening?